requirements_dev.txt

```diff
@@ -14,3 +14,5 @@ coverage
 
 # For building DEs
 ansible-builder
+
+pre-commit
```

Comment:
If we add pre-commit to the requirements, [this section of the docs](https://github.com/ansible/ansible-rulebook/blob/main/docs/development_environment.rst#git-pre-commit-hooks-optional) needs to be updated as well because we no longer need to install it manually.